### PR TITLE
Create Citation

### PR DESCRIPTION
**Pull request by VartanAhrensKayayan**
Only cited one of the main contributors but altering the document is a simple task. If the other contributors want to be cited I can make the changes.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "If you use this software, please cite it as
+  below."
+type: software
+authors:
+  - given-names: Jan
+    family-names: Pecinovsky
+    email: janpecinovsky@gmail.com
+    affiliation: EnergieID
+  - given-names: Frank
+    family-names: Boermand
```
Comment (fboerman) [inline CITATION.cff:13]: my name is Boerman without the d :P

**Comment by fboerman** [comment]
hi @VartanAhrensKayayan thanks for providing this. if you fix my name I will merge it

**Comment by VartanAhrensKayayan** [comment]
@fboerman, it should be correct now. Mea culpa.

**Comment by fboerman** [comment]
great thanks!

**Pull request merged**